spark-cassandra-connector-java/src/main/java/com/datastax/spark/connector/japi/rdd/CassandraJavaRDD.java

```diff
@@ -110,4 +110,13 @@ public CassandraJavaRDD(CassandraRDD<R> rdd, ClassTag<R> classTag) {
     public <K> JavaPairRDD<K, Iterable<R>> spanBy(Function<R, K> f, ClassTag<K> keyClassTag) {
         return CassandraJavaUtil.javaFunctions(rdd()).spanBy(f, keyClassTag);
     }
+
+    /**
+     * Produces the empty CassandraRDD which has the same signature and properties, but it does not
+     * perform any validation and it does not even try to return any rows.
+     */
+    public CassandraJavaRDD<R> toEmptyCassandraRDD() {
+        CassandraRDD<R> newRDD = rdd().toEmptyCassandraRDD();
```

Comment:
This should stay in line with in the spark API:

      def makeRDD[T: ClassTag](seq: Seq[T], numSlices: Int = defaultParallelism): RDD[T]  

So:

    def makeCassandraRDD[T: ClassTag](seq: Seq[T], numSlices: Int = defaultParallelism): CassandraRDD[T]